utilities.js

```diff
@@ -21,7 +21,6 @@ export async function * combineAsyncIterators(iterator1, iterator2) {
 }
 
 export async function * lineIterator(stream) {
-	stream.setEncoding('utf8');
```

Comment:
This function is only called when the user iterates over `promise.stdout` or `promise.stderr`. But `result.stdout`/`result.stderr` always need the stream encoding to be `utf8`.